tests/snappy_pipeline/workflows/test_workflows_cbioportal_export.py

```diff
@@ -102,10 +75,9 @@ def cbioportal_export_workflow(
     # Patch out file-system related things in abstract (the crawling link in step is defined there)
     patch_module_fs("snappy_pipeline.workflows.abstract", cancer_sheet_fake_fs, mocker)
     dummy_workflow.globals = {
-        "ngs_mapping": lambda x: "NGS_MAPPING/" + x,
-        "copy_number_step": lambda x: "SOM_CNV_CALLING/" + x,
-        "somatic_variant_filtration": lambda x: "SOM_VAR_FILTRATION/" + x[0],
-        "gene_expression_quantification": lambda x: "GENE_EXP_QUANTIFICATION/" + x,
+        "ngs_mapping": lambda x: "/NGS_MAPPING/" + x,
+        "somatic_variant": lambda x: "/SOM_VAR_FILTRATION/" + x,
+        "copy_number_step": lambda x: "/COPY_NUMBER" + x,
```

Comment:
Is there a "/" missing after "/COPY_NUMBER"?

Reply:
Yes, there should be. But for some reason that I don't understand, the tests are passed whether or not the `/` is added...